flow/result.go

```diff
@@ -107,16 +107,16 @@ func GetBlockEvents(host string, height uint64, eventType string) {
 
 	events, err := flowClient.GetEventsForHeightRange(ctx, client.EventRangeQuery{
 		Type:        eventType,
-		StartHeight: height,
-		EndHeight:   height,
+		StartHeight: startHeight,
+		EndHeight:   endHeight,
 	})
 
 	if err != nil {
 		Exitf(1, "Failed to query block event by height: %s", err)
 	}
 
-	for _, blockEvent := range events {
-		fmt.Printf("Events for Block %s:", blockEvent.BlockID)
+	for i, blockEvent := range events {
+		fmt.Printf("Events for Block %s (%d):", blockEvent.BlockID, startHeight+uint64(i))
```

Comment:
What does the number in parenthesis represent?

Reply:
it's the height of the block. better to label?